cmd/dstask.go

```diff
@@ -214,25 +214,32 @@ func main() {
 			dstask.MustGitCommit("Edited %s", task)
 		}
 
-	case dstask.CMD_NOTES:
-		fallthrough
-	case dstask.CMD_NOTE:
+	case dstask.CMD_NOTE, dstask.CMD_NOTES:
```

Comment:
Hm, seems I have added aliases before! The `rm`/`remove` thing is fine by precedent, for now then.